telegramgallery/src/main/java/com/tangxiaolv/telegramgallery/PhotoAlbumPickerActivity.java

```diff
@@ -162,6 +161,11 @@ public void onItemClick(int id) {
 
             ActionBarMenu menu = actionBar.createMenu();
             dropDownContainer = new ActionBarMenuItem(context, menu, 0);
+            dropDownContainer.setSubMenuOpenSide(1);
```

Comment:
@GuillaumeDelente Update brought this in, I kept it in since we will be making more changes here to support photo + video